specification/src/chapter2.adoc

```diff
@@ -537,7 +622,10 @@ Running unauthorized software can compromise the security state of the system.
 | Requirement
 
 | CAT_NNN
-| A system in secured state MUST only load authorized software.
+| A system in secured or recoverable debug states MUST only load authorized software.
+
+| CAT_NNN
+| A system in secure provisioning state SHOULD only load authorized software.
```

Comment:
```suggestion
| A system in secure provisioning state SHOULD only load cryptographically authenticated software.
```

Reply:
The terms "authorized software" "measured software" and "verified software" are defined in the ecosystem chapter. I don't think we need to introduce new terms here.